feature/home/src/main/java/com/epmedu/animeal/home/presentation/thankyou/ThankYouScreenUi.kt

```diff
@@ -0,0 +1,89 @@
+package com.epmedu.animeal.home.presentation.thankyou
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.epmedu.animeal.foundation.button.AnimealButton
+import com.epmedu.animeal.foundation.preview.AnimealPreview
+import com.epmedu.animeal.foundation.theme.AnimealTheme
+import com.epmedu.animeal.resources.R
+
+@Composable
+internal fun ThankYouScreenUI(
+    modifier: Modifier = Modifier,
+    onBackToHome: () -> Unit
+) {
+    Column(
+        modifier = modifier.fillMaxSize(),
+        verticalArrangement = Arrangement.SpaceBetween
+    ) {
+        Row(
+            modifier = Modifier.weight(1f),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Thanks()
+        }
+        BackButton(onBackToHome)
+    }
+}
+
+@Composable
+fun Thanks() {
```

Comment:
Can you please rename it to ThankYouContent, move the composable to thankyou.ui package and make it internal to follow consistency in our project?